.github/ci_setup.sh

```diff
@@ -0,0 +1,30 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+#
+# This script is meant to configure the build/runtime environment of the
+# Docker contaners that are used in the project's CI configuration.
+#
+# Usage: source .github/ci_setup.sh <platform name>
+#
+
+# The platform name.
+PLATFORM_NAME=$1
+
+# Set up the correct environment for the SYCL tests.
+if [ "${PLATFORM_NAME}" = "SYCL" ]; then
+   source /opt/intel/oneapi/setvars.sh
```

Comment:
You should in general simplify this script. Have a look at: https://github.com/acts-project/vecmem/blob/main/.github/ci_setup.sh

With the latest versions of the Docker images much of the environment setup is done automatically, so this setup script can be a whole lot simpler.

Reply:
Updated